### DISCUSSION TITLE
Spectral Data Sets

### DISCUSSION THREAD

**Discussion by brianbourke75** [whitews/FlowKit#228] [General]
Hiya!  Thanks for the 1.2.1 release, it's working great for me so far!  I have a question regarding spectral data.  The compensation matrix is obviously different from conventional data sets and the sample file does not require compensation.  When reading in data sets, how would you suggest skipping the compensation step so that I can still gate the events?  Thanks!

**Comment by whitews** [comment]
Hey Brian,

Good to hear the latest release is working well. So the compensation & transform steps are independent AND optional for any `Dimension` in any gate. If you want to skip the compensation step, just use the value "uncompensated" for the `compensation_ref` kwarg (it's the default value so you don't even need to specify it). You can read more in [the tutorial on the `gates` module](https://flowkit.readthedocs.io/en/latest/notebooks/flowkit-tutorial-part04-gates-module.html#Dimension-Class).

Regards,
Scott

**Reply by brianbourke75** [reply]
Thanks for the feedback, working like a charm!
